supplemental_content/views.py

```diff
@@ -42,3 +45,38 @@ def get_queryset(self):
                 )
             ).distinct()
         return query
+
+
+class SupplementalContentSectionsView(generics.CreateAPIView):
```

Comment:
What kind of permissions does this endpoint have?  We can't have just anyone posting these to our URL.

Reply:
There was no auth. I just added it in.